mteb/tasks/Retrieval/eng/ClimateFEVERRetrieval.py

```diff
@@ -72,3 +72,39 @@ class ClimateFEVERHardNegatives(AbsTaskRetrieval):
       primaryClass={cs.CL}
 }""",
     )
+
+
+class ClimateFEVERv2(AbsTaskRetrieval):
```

Comment:
You will need to add `supeseeded_by` to ClimateFEVER

If we want to consistently name tasks we should probably call this

```suggestion
class ClimateFEVERRetrievalv2(AbsTaskRetrieval):
```

The same with the name